packages/business-rules/src/schemas/householder-appeal/insert.js

```diff
@@ -100,6 +100,24 @@ const insert = pinsYup
         })
         .noUnknown(true),
     }),
+    contactDetailsSection: pinsYup
+      .object()
+      .shape({
+        name: pinsYup.lazy((name) => {
+          if (name) {
+            return pinsYup
+              .string()
+              .min(2)
+              .max(80)
+              .matches(/^[a-z\-' ]+$/i)
+              .required();
+          }
+          return pinsYup.string().nullable();
+        }),
```

Comment:
Is the reason that `name` is in the conditional in the schemas because you found that it's not possible to use `matches()` and `nullable()` together?